tests/functests/test_suite_test.go

```diff
@@ -0,0 +1,83 @@
+package tests
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	core "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/tools/clientcmd"
+	"kubevirt.io/client-go/kubecli"
+)
+
+const (
+	testNamespace = "common-instancetype-functest"
+)
+
+var (
+	virtClient kubecli.KubevirtClient
+)
+
+func deleteBundles() {
+	virtualMachineClusterInstancetypes, err := virtClient.VirtualMachineClusterInstancetype().List(context.Background(), metav1.ListOptions{})
+	Expect(err).ToNot(HaveOccurred())
+	for _, instancetype := range virtualMachineClusterInstancetypes.Items {
+		err := virtClient.VirtualMachineClusterInstancetype().Delete(context.Background(), instancetype.Name, metav1.DeleteOptions{})
+		Expect(err).ToNot(HaveOccurred())
+	}
+
+	virtualMachineClusterPreferences, err := virtClient.VirtualMachineClusterPreference().List(context.Background(), metav1.ListOptions{})
+	Expect(err).ToNot(HaveOccurred())
+	for _, preference := range virtualMachineClusterPreferences.Items {
+		err := virtClient.VirtualMachineClusterPreference().Delete(context.Background(), preference.Name, metav1.DeleteOptions{})
+		Expect(err).ToNot(HaveOccurred())
+	}
+}
+
+var _ = BeforeSuite(func() {
+	kubeconfigPath := os.Getenv(clientcmd.RecommendedConfigPathEnvVar)
+	config, err := clientcmd.BuildConfigFromFlags("", kubeconfigPath)
+	Expect(err).ToNot(HaveOccurred())
+
+	virtClient, err = kubecli.GetKubevirtClientFromRESTConfig(config)
+	Expect(err).ToNot(HaveOccurred())
+
+	namespaceObj := &core.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: testNamespace,
+		},
+	}
+	_, err = virtClient.CoreV1().Namespaces().Create(context.TODO(), namespaceObj, metav1.CreateOptions{})
+	Expect(err).ToNot(HaveOccurred())
+
+	// Check if resorces are created
+	virtualMachineClusterInstancetypes, err := virtClient.VirtualMachineClusterInstancetype().List(context.Background(), metav1.ListOptions{})
+	Expect(err).ToNot(HaveOccurred())
+	Expect(len(virtualMachineClusterInstancetypes.Items)).ToNot(BeZero())
+
+	virtualMachineClusterPreferences, err := virtClient.VirtualMachineClusterPreference().List(context.Background(), metav1.ListOptions{})
+	Expect(err).ToNot(HaveOccurred())
+	Expect(len(virtualMachineClusterPreferences.Items)).ToNot(BeZero())
+})
+
+var _ = AfterSuite(func() {
+	// Clean up namespaced resources
+	err := virtClient.CoreV1().Namespaces().Delete(context.TODO(), testNamespace, metav1.DeleteOptions{})
+	if err != nil && !errors.IsNotFound(err) {
+		Expect(err).ToNot(HaveOccurred())
+	}
+
+	// Cluster type resources needs to be removed explicitly
+	// Clean up after running kubevirt-sync
+	deleteBundles()
```

Comment:
Not needed, user can throw away the cluster themselves. Or maybe they'd like to keep the deployed resources?